package.json

```diff
@@ -116,7 +116,7 @@
     "start:companion": "bash bin/companion.sh",
     "start:companion:with-loadbalancer": "e2e/start-companion-with-load-balancer.mjs",
     "build:bundle": "yarn node ./bin/build-bundle.mjs",
-    "build:clean": "rm -rf packages/*/lib packages/@uppy/*/lib packages/*/dist packages/@uppy/*/dist",
+    "build:clean": "git clean -e node_modules -xfd packages e2e .parcel-cache coverage",
```

Comment:
seems a bit much to clean node_modules? Especially for every e2e run.

Reply:
`-e` is for `exclude`: https://git-scm.com/docs/git-clean#Documentation/git-clean.txt--eltpatterngt